src/transformers/pipelines/conversational.py

```diff
@@ -333,6 +372,9 @@ def _concat_inputs_history(self, inputs: List[List[int]], histories: List[Option
                     if cutoff_eos_index == 0 or cutoff_eos_index == len(new_input) - 1:
                         break
                     else:
+                        logger.warning(
+                            "Cutting history off because it's too long ({len(new_input)} > {max_length - self.min_length_for_response}) for underlying model"
```

Comment:
```suggestion
                            f"Cutting history off because it's too long ({len(new_input)} > {max_length - self.min_length_for_response}) for underlying model"
```
Seems to be an f-string.